src/db/localMongooseConnectionDb.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+module.exports = {
+  connect: () => {
+    mongoose.Promise = global.Promise;
+    const options = {
+      keepAlive: true,
+      socketTimeoutMS: 540000,
+      autoReconnect: true,
+      reconnectTries: Number.MAX_VALUE,
+      autoIndex: false,
+      reconnectInterval: 500,
+      poolSize: 10,
+      bufferMaxEntries: 0,
+      useNewUrlParser: true,
+    };
+
+    if (process.env.ENVIRONMENT === 'dev') {
```

Comment:
Essa configuração de ambiente é exclusiva para o mongo? Se não for o padrão é colocar no pacote config, do mesmo jeito que está sendo feito em todos os outros repositórios.

Reply:
Acredito ser exclusiva para o mongo, pois os outros microsserviços seguem esse mesmo padrão, porém neles existem a pasta environment, que trata das variáveis globais.